cocos/2d/CCNode.cpp

```diff
@@ -1797,6 +1798,12 @@ void Node::setAdditionalTransform(Mat4* additionalTransform)
     _transformUpdated = _transformDirty = _inverseDirty = true;
 }
 
+void Node::setAdditionalTransform(const Mat4& additionalTransform)
+{
+    _useAdditionalTransform = true;
+    _additionalTransform = additionalTransform;
+    _transformUpdated = _transformDirty = _inverseDirty = true;
+}
```

Comment:
yes, in this case, actually it is better to use the newly added function since it passes a const reference. Most of our APIs are expecting const references instead of pointers.
And in this case in particular, it is very handy to use it. since the function that calls it expects also a const reference.